resources/views/bread/read.blade.php

```diff
@@ -180,4 +180,5 @@
         });
 
     </script>
+    @includeWhen($dataTypeContent->getCoordinates(), 'voyager::formfields.coordinates_js')
```

Comment:
Method `getCoordinates` does not exist according to test `TCG\Voyager\Tests\RouteTest::testGetRoutes`.

> ErrorException: Call to undefined method Illuminate\Database\Query\Builder::getCoordinates() (View: /home/travis/build/the-control-group/voyager/resources/views/bread/read.blade.php)

Reply:
That's because it's acting on a `Builder` object, not a `Model` object.  Likely missing a `->get()` somewhere